.slackbot.yml

```diff
@@ -0,0 +1,15 @@
+releaseSchedule:
+  ggetz: 2/1/2019
+  hpinkos: 3/1/2019
+  lilleyse: 4/1/2019
+  gary: 5/1/2019
+  dbagnell: 6/3/2019
+  mamato: 7/1/2019
+  tfili: 8/1/2019
+
+greetings:
+  - Happy Friday everyone!
```

Comment:
Why is this list using `-` but not the `releaseSchedule` list?  And what is this used for?

Reply:
This is just YAML syntax because `releaseSchedule` is a dictionary and `greetings` is an array. The Concierge slackbot uses this in its weekly message, see https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/131